modules/swap/swap.tf

```diff
@@ -0,0 +1,30 @@
+resource "datadog_monitor" "swap_free" {
+  count = "${var.monitor_enabled ? 1 : 0}"
+  name  = "Insufficient of free swap ${module.label.id}"
+  type  = "${var.alert_type}"
+
+  message = <<EOF
```

Comment:
What's the behavior of TF HEREDOCs? Will it strip off these leading spaces? If not, we should consider moving this to a template file.